gradle.properties

```diff
@@ -1,4 +1,5 @@
 findapet.environment=prod
+findapet.petfinder.apikey=23bfea78dfb8bde56bbae16192cbe6d4
```

Comment:
Will move to something local and not in a repo for production (if that ever happens) of course. Not worrying about it for now, so it's easier for anyone to build the app themselves